static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -0,0 +1,145 @@
+import {forwardRef as reactFowardRef, useEffect, useState} from 'react';
+
+import Input from 'sentry/components/forms/controls/input';
+import Slider from 'sentry/components/forms/controls/rangeSlider/slider';
+import SliderAndInputWrapper from 'sentry/components/forms/controls/rangeSlider/sliderAndInputWrapper';
+import SliderLabel from 'sentry/components/forms/controls/rangeSlider/sliderLabel';
+import type SliderProps from 'sentry/components/forms/controls/rangeSlider/sliderProps';
```

Comment:
Since these are all relative I would probably do it like this

```suggestion
import Slider from './slider';
import SliderAndInputWrapper from './sliderAndInputWrapper';
import SliderLabel from './sliderLabel';
import type SliderProps from './sliderProps';
```